src/ai-store/generative-ai-service.js

```diff
@@ -290,7 +290,7 @@ class BrowserGenerativeAiService extends GenerativeAiService {
 			}
 		}
 
-		const session = await window.ai.createTextSession( modelParams );
+		const session = await self.ai.assistant.prompt( modelParams );
```

Comment:
Does this need to be `create` instead of `prompt`, per https://github.com/explainers-by-googlers/prompt-api/tree/main?tab=readme-ov-file#emulating-tool-use-or-function-calling-via-assistant-role-prompts?

As far as I understand, this creates the session and then you pass the prompt via the session's `prompt` method below.

```suggestion
		const session = await self.ai.assistant.create( modelParams );
```

Reply:
Duh, what was I thinking? Of course `create()`.